docs/GettingStartedDocs/Contributors/SGX1FLCGettingStarted.md

```diff
@@ -30,13 +30,13 @@ sudo scripts/ansible/install-ansible.sh
 If you are running in an Azure Confidential Compute (ACC) VM and would like to use the attestation features, you should also run the following command from the root of the source tree:
 
 ```bash
-sudo ansible-playbook scripts/ansible/oe-contributors-acc-setup.yml
```

Comment:
Are we sure sudo is not required? I was pretty sure it was.

Reply:
Yes, I am sure.

The playbooks `oe-contributors-acc-setup.yml`,  `oe-contributors-setup.yml`, and `oe-contributors-setup-sgx1.yml` have the `become: yes` flag when executing their tasks:
* https://github.com/openenclave/openenclave/blob/574ba3b0f0ca9eddc3d45ed72a0df472bc1fd60b/scripts/ansible/oe-contributors-acc-setup.yml#L7
* https://github.com/openenclave/openenclave/blob/574ba3b0f0ca9eddc3d45ed72a0df472bc1fd60b/scripts/ansible/oe-contributors-setup.yml#L7
* https://github.com/openenclave/openenclave/blob/574ba3b0f0ca9eddc3d45ed72a0df472bc1fd60b/scripts/ansible/oe-contributors-setup-sgx1.yml#L7

Thus, the playbooks execute with `sudo` rights. However, if the user has a password for `sudo` (which doesn't happen on our testing VMs), the following extra parameter `--ask-become-pass` must be passed to the `ansible-playbook` command. I'll document this.

Also, I found out that if you execute the `ansible-playbook` command with `sudo` on RHEL 8 platform, by default, the `PATH` from the `root` user doesn't have the directory with the `ansible-playbook` binary, leading to `command not found` kind of error.

So, I think it's easier to have the documentation like this: execute `ansible-playbook` without `sudo`, and make sure you pass the `--ask-become-pass` flag if sudo has a password.

Reply:
This makes a lot of sense, thanks for clarifying, I agree.